spec/forms/enterprise_fees_bulk_update_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe EnterpriseFeesBulkUpdate do
+  describe "error reporting" do
+    let(:enterprise_fee) { build_stubbed(:enterprise_fee) }
```

Comment:
Haha, I've been learning a lot through this, which is good because I just got a job refactoring/specing controllers for a mature app. Thanks for the feedback and help on this!

Reply:
That's great, and congratulations! Sounds like the job will be a good fit.